<?xml version="1.0" encoding="utf-8"?>
<ui version="4.0">
 <author>
    SPDX-FileCopyrightText: none
    SPDX-License-Identifier: GPL-3.0-or-later
  </author>
 <class>HalftoneConfigPageWidget</class>
 <widget class="QWidget" name="HalftoneConfigPageWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>376</width>
    <height>394</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <property name="usesScrollButtons">
      <bool>false</bool>
     </property>
     <widget class="QWidget" name="tabScreenGenerator">
      <attribute name="title">
       <string>Screen Generator</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout" stretch="0,1">
       <property name="spacing">
        <number>5</number>
       </property>
       <property name="leftMargin">
        <number>10</number>
       </property>
       <property name="topMargin">
        <number>10</number>
       </property>
       <property name="rightMargin">
        <number>10</number>
       </property>
       <property name="bottomMargin">
        <number>10</number>
       </property>
       <item>
        <widget class="QComboBox" name="comboBoxGenerator">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widgetGeneratorContainer" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabPostprocessing">
      <attribute name="title">
       <string>Postprocessing</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,1">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>10</number>
       </property>
       <property name="topMargin">
        <number>10</number>
       </property>
       <property name="rightMargin">
        <number>10</number>
       </property>
       <property name="bottomMargin">
        <number>10</number>
       </property>
       <item>
        <layout class="QFormLayout" name="formLayout_2">
         <property name="horizontalSpacing">
          <number>5</number>
         </property>
         <property name="verticalSpacing">
          <number>5</number>
         </property>
         <item column="0" row="0">
          <widget class="QLabel" name="labelHardness">
           <property name="text">
            <string>Hardness:</string>
           </property>
          </widget>
         </item>
         <item column="1" row="0">
          <widget class="KisDoubleSliderSpinBox" name="sliderHardness" native="true">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item column="0" row="1">
          <widget class="QLabel" name="labelInvert">
           <property name="text">
            <string>Invert:</string>
           </property>
          </widget>
         </item>
         <item column="1" row="1">
          <widget class="QCheckBox" name="checkBoxInvert">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item column="0" row="2">
          <widget class="QLabel" name="labelForeground">
           <property name="text">
            <string>Foreground:</string>
           </property>
          </widget>
         </item>
         <item column="1" row="2">
          <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,2">
           <property name="spacing">
            <number>5</number>
           </property>
           <item>
            <widget class="KisColorButton" name="buttonForegroundColor">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="KisSliderSpinBox" name="sliderForegroundOpacity" native="true">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item column="0" row="3">
          <widget class="QLabel" name="labelBackground">
           <property name="text">
            <string>Background:</string>
           </property>
          </widget>
         </item>
         <item column="1" row="3">
          <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="1,2">
           <property name="spacing">
            <number>5</number>
           </property>
           <item>
            <widget class="KisColorButton" name="buttonBackgroundColor">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="KisSliderSpinBox" name="sliderBackgroundOpacity" native="true">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KisSliderSpinBox</class>
   <extends>QWidget</extends>
   <header location="global">kis_slider_spin_box.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>KisColorButton</class>
   <extends>QPushButton</extends>
   <header location="global">kis_color_button.h</header>
  </customwidget>
  <customwidget>
   <class>KisDoubleSliderSpinBox</class>
   <extends>QWidget</extends>
   <header location="global">kis_slider_spin_box.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>